src/app/features/home/post-capture-tab/post-capture-tab.component.scss

```diff
@@ -13,3 +13,107 @@ mat-grid-tile {
     border-radius: 4px;
   }
 }
+
+.sub-content {
+  padding-top: 60px;
+}
+
+.tab-content-post {
+  virtual-scroller {
+    width: 100vw;
+    height: 93vh;
+    top: 7vh;
+  }
+
+  app-post-capture-card {
+    width: 100%;
+  }
+}
+
+.capture-container {
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.set-container {
+  display: flex;
+  width: 100%;
+  height: 162px;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  margin-top: 10px;
+  margin-bottom: 20px;
+}
+
+.set-background {
+  display: flex;
+  padding-right: 5px;
+  background: white;
+  box-sizing: border-box;
+  box-shadow: 4px 0 4px rgb(0 0 0/ 0.25);
+  border-radius: 5px;
+}
+
+.set-image {
+  width: 350px;
```

Comment:
Why do we need to hard code the width of the series component? These magic numbers would be really confusing in the future.